coffee_core/src/coffee.rs

```diff
@@ -195,10 +196,24 @@ impl CoffeeManager {
         if self.config.cln_config_path.is_none() {
             return Ok(());
         }
-        let root = self.config.cln_root.clone().unwrap();
+        let root = self
+            .config
+            .cln_root
+            .clone()
+            .ok_or_else(|| error!("cln root path not found, please run `coffee setup`"))?;
```

Comment:
```suggestion
        let root = self
            .config
            .cln_root;
       let root = root
            .ok_or_else(|| error!("cln root path not found, please run `coffee setup`"))?;
```

Avoid long chain, it is huggly

Reply:
Fixed